ui/de-lib/src/main/java/org/iplantc/de/client/services/AppUserServiceFacade.java

```diff
@@ -82,4 +82,12 @@ void rateApp(App app,
                  AsyncCallback<AppFeedback> callback);
 
     void deleteRating(App app, AsyncCallback<AppFeedback> callback);
+
+    void getPermissions(List<App> apps, AsyncCallback<String> callback);
```

Comment:
It seems like this method should be updated as well;
`void getPermissions(List<App> apps, AsyncCallback<Map<String, List<Sharing>> callback);`
Then the logic in the `AppSharingPresenter.LoadPermissionsCallback.GetUserInfoCallback.onSuccess()` can be moved into a callback converter in `AppUserServiceFacadeImpl.getPermissions(..)`.

This also holds true for `DiskResourceServiceFacade.getPermissions(..)` except a little more;
`void getPermissions(HasPaths diskResourceIds, AsyncCallback<Map<String, List<Sharing>> callback);`
And the logic to construct the request will come from `DataSharingPresenterImpl.buildPermissionsRequestBody()`.